src/interactions/morse.jl

```diff
@@ -0,0 +1,32 @@
+export Morse
+
+"""
+    Morse
+
+Morse potential for bonds between two atoms
+
+"""
+
+struct Morse{T, A, R} <: SpecificInteraction
+    D::T
+    α::A
+    r0::R
+end
+
```

Comment:
Add a constructor using keyword arguments here like for `HarmonicBond`.

Reply:
Forgot that initially. Took care of it, it's done.